src/pages/workspace/WorkspaceCardPage.js

```diff
@@ -0,0 +1,187 @@
+                title={translate('workspace.common.card')}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <ScrollView style={[styles.settingsPageBackground]} bounces={false}>
+                <View style={styles.pageWrapper}>
+                    <View style={[
+                        styles.mb3,
+                        styles.flexRow,
+                        styles.workspaceCard,
+                        isSmallScreenWidth && styles.workspaceCardMobile,
+                    ]}
+                    >
+                        {isSmallScreenWidth
+                            ? (
+                                <HeroCardMobileImage
+                                    style={StyleSheet.flatten([styles.fullscreenCard, styles.fullscreenCardMobile])}
+                                />
+                            )
+                            : (
+                                <HeroCardWebImage
+                                    style={StyleSheet.flatten([styles.fullscreenCard, styles.fullscreenCardWeb])}
+                                />
+                            )}
+
+                        <View style={[
+                            styles.fullscreenCard,
+                            styles.workspaceCardContent,
+                            isSmallScreenWidth && styles.p5,
+                        ]}
+                        >
+                            <View
+                                style={[
+                                    styles.flexGrow1,
+                                    styles.justifyContentEnd,
+                                    styles.alignItemsStart,
+                                    !isSmallScreenWidth && styles.w50,
+                                ]}
+                            >
+                                <Text
+                                    style={[
+                                        styles.workspaceCardMainText,
+                                        styles.mb5,
+                                    ]}
+                                    color={themeDefault.textReversed}
+                                >
+                                    {user.isUsingExpensifyCard
+                                        ? translate('workspace.card.cardReadyTagline')
+                                        : translate('workspace.card.tagline')}
+                                </Text>
+                                <Text
+                                    fontSize={variables.fontSizeLarge}
+                                    color={themeDefault.textReversed}
+                                    style={[styles.mb8]}
+                                >
+                                    {user.isFromPublicDomain
+                                        ? (
+                                            <>
+                                                <RNText>{translate('workspace.card.publicCopy')}</RNText>
```

Comment:
Why are we using `RNText` here instead of `Text`?

Reply:
So that it inherits the base Text style from the Parent Text Component. Otherwise cascading does not work.